exhibitor-standalone/src/main/java/com/netflix/exhibitor/application/ExhibitorMain.java

```diff
@@ -124,7 +126,16 @@ public ExhibitorMain(BackupProvider backupProvider, ConfigProvider configProvide
 
         DefaultResourceConfig   application = JerseySupport.newApplicationConfig(new UIContext(exhibitor));
         ServletContainer        container = new ServletContainer(application);
-        server = new Server(httpPort);
+
+        SelectChannelConnector connector = new SelectChannelConnector();
```

Comment:
In my ab tests I saw Exhibitor go up to about 4k open file descriptors. Would like to figure out how to limit that explicitly.